org.standardnotes.standardnotes.yml

```diff
@@ -0,0 +1,163 @@
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
+        sha256: b62c0e7937551d0cc02b8fd5cb0f544f9405bafc9a54d3808ed4594812edef43
+
+  - name: standardnotes
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: '/usr/lib/sdk/node14/bin:/app/yarn/bin:/run/build/standardnotes/flatpak-node/chromedriver/'
+      env:
+        # Don't add ELECTRON_CACHE
+        XDG_CACHE_HOME: /run/build/standardnotes/flatpak-node/cache
+        # ELECTRON_CACHE: '/run/build/standardnotes/flatpak-node/electron-cache/'
+        CHROMEDRIVER_SKIP_DOWNLOAD: 'true'
+        JOBS: max
+        npm_config_nodedir: /run/build/standardnotes/flatpak-node/cache/node-gyp/9.4.2
+    build-commands:
+      # Retire this at some point; (1) create a new task by default and (2) use Yarn instead"
+      - "sed -i '/\\\"pack\\\"/c\\    \\\"pack:linux\\\": \\\"npm run
+        build:linux && electron-builder --linux\\\",' package.json"
+      # No need to prettify the code
+      - sed -i -e "s/npm run lint:formatting && npm run lint:eslint && //"
+        package.json
+      # Remove lockfiles used by npm
+      - "find . -type f -name 'package-lock.json' -exec rm {} +"
+
+      # Use the offline mirror.
+      - 'HOME=$PWD yarn config --offline set yarn-offline-mirror
+        $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror'
+
+      # Translated from npm run setup
+      - yarn --offline --pure-lockfile
+      - yarn --offline --pure-lockfile --cwd ./app
+      - yarn --offline --pure-lockfile --cwd ./web
+
+      # - 'npm --offline --save false ci
+      #   --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache'
+      # - 'npm --offline --save false --prefix ./web install
+      #   --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache'
+      # - 'npm --offline --save false --prefix ./app ci
+      #   --cache=$FLATPAK_BUILDER_BUILDDIR/flatpak-node/npm-cache'
+
+      - . flatpak-node/electron-builder-arch-args.sh; yarn run --offline
+        build:linux -- $ELECTRON_BUILDER_ARCH_ARGS
+      # Copy the resulting, unpacked directory to /app.
+      # (A glob is used because the directory name may contain the current arch.)
+      - 'cp -r standardnotes/dist/linux/*/linux*unpacked /app/standardnotes'
+
+      # Install the wrapper script to start it.
+      - 'install -Dm 755 start-standardnotes.sh /app/bin/start-standardnotes'
+
+      # Retire this at some point when an SVG is available
+      - install -Dm644 app/icon/Icon-256x256.png
+        /app/share/icons/hicolor/256x256/apps/org.standardnotes.standardnotes.png
+      - install -Dm644 app/icon/Icon-512x512.png
+        /app/share/icons/hicolor/512x512/apps/org.standardnotes.standardnotes.png
+    sources:
+      - type: file
+        path: yarn.lock
+      - type: file
+        path: yarn.app.lock
+        dest-filename: yarn.lock
```

Comment:
Actually, you can use npm to install ./ and ./app,
and use yarn to install ./web,
and generate sources for those two separately.
That's avoid manually add yarn.lock.

Reply:
You're right. After all it's only for dependency resolution